torch/csrc/distributed/rpc/message.h

```diff
@@ -115,6 +115,14 @@ class TORCH_API Message final {
   int64_t id_ = -1;
 };
 
+// create an exception given a message and exception.
+TORCH_API Message
+createException(const Message& request, const std::exception& e);
```

Comment:
nit: Maybe we should call this `createExceptionResponse` and clearly state in the docs that we're building a `response` Message with an exception for the provided `request` Message.